lib/index.js

```diff
@@ -1,29 +1,68 @@
 'use strict';
 
+var pg = require('pg');
 var addDefaultValues = require('./add_default_values.js');
 var loadSchema = require('./load_schema.js');
 var updateSchema = require('./update_schema.js');
+var routeGen = require('./helpers/endpoints_helpers/endpoints_generator.js');
+var configurationRoutes = require('./helpers/configuration/configuration.js');
+var db = require('./db.js');
+
+require('env2')('config.env');
 
 module.exports = function (server, options, next) {
+  var routes;
+
+  if (!process.env.DATABASE_URL) {
+    return next(new Error('DATABASE_URL environment variable is undefined'));
+  }
+
+  // server.app.db = new pg.Pool(process.env.DATABASE_URL);
+  server.app.db = new pg.Pool({ database: 'abase_test' });
```

Comment:
Does the connection work by just passing the url of the database? I haven't manage to make it work on my ubuntu with just the url, I had to convert it to an object. It migt be good to test it on heroku to see if it's a problem there too.

from pg-pool https://github.com/brianc/node-pg-pool:
The Pool constructor does not support passing a Database URL as the parameter. To use pg-pool on heroku, for example, you need to parse the URL into a config object. Here is an example of how to parse a Database URL.